Client/Frontend/Widgets/ActionOverlayTableViewController.swift

```diff
@@ -60,7 +61,8 @@ class ActionOverlayTableViewController: UIViewController, UITableViewDelegate, U
         tableView.snp_makeConstraints { make in
             make.center.equalTo(self.view)
             make.width.equalTo(290)
-            make.height.equalTo(74 + actions.count * 56)
+            make.height.lessThanOrEqualTo(UIScreen.mainScreen().bounds.size.height).priorityHigh()
+            make.height.equalTo(74 + actions.count * 56).priorityLow()
```

Comment:
Make a UX struct and add these hardcoded values and access them like so: `ActionTableViewControllerUX.height + actions.count....` etc.